orchagent/orchdaemon.cpp

```diff
@@ -66,7 +66,11 @@ bool OrchDaemon::init()
 
     gCrmOrch = new CrmOrch(m_configDb, CFG_CRM_TABLE_NAME);
     gPortsOrch = new PortsOrch(m_applDb, ports_tables);
-    gFdbOrch = new FdbOrch(m_applDb, APP_FDB_TABLE_NAME, gPortsOrch);
+
+    TableConnector applDbFdb(m_applDb, APP_FDB_TABLE_NAME);
+    TableConnector stateDbFdb(m_stateDb, APP_FDB_TABLE_NAME);
```

Comment:
APP_FDB_TABLE_NAME for stateDB?

Reply:
APP_FDB_TABLE_NAME is a table name. FDB_TABLE_NAME may be a better name. I admitted the name is confusing and may refine it in future.

---
In reply to: [217208443](https://github.com/Azure/sonic-swss/pull/615#discussion_r217208443) [](ancestors = 217208443)

Reply:
Resolved in this PR.

---
In reply to: [217835174](https://github.com/Azure/sonic-swss/pull/615#discussion_r217835174) [](ancestors = 217835174,217208443)